bin/lib/migrateLegacyResources.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs')
+const Path = require('path')
+const promisify = require('util').promisify
+
+/* Converts the old (pre-5.0.0) extensionless files to $-based files _with_ extensions
+ * to make them work in the new resource mapper (post-5.0.0).
+ * By default, all extensionless files (that used to be interpreted as Turtle) will now receive a '$.ttl' suffix. */
+/* https://www.w3.org/DesignIssues/HTTPFilenameMapping.html */
+
+module.exports = function (program) {
+  program
+    .command('migrate-legacy-resources')
+    .option('-p, --path <path>', 'Path to the data folder, defaults to \'data/\'')
+    .option('-s, --suffix <path>', 'The suffix to add, defaults to \'$.ttl\'')
+    .option('-v, --verbose', 'Path to the data folder')
+    .description('Migrate extensionless data files pre-5.0.0 to turtle-based data files post-5.0.0')
```

Comment:
"Migrate the data folder from node-solid-server 4 to node-solid-server 5" or similar>